internal/job/shell/shell_test.go

```diff
@@ -335,24 +349,27 @@ func acquireLockInOtherProcess(lockfile string) (*exec.Cmd, error) {
 
 // TestAcquiringLockHelperProcess isn't a real test. It's used as a helper process
```

Comment:
Is this comment still correct?

Reply:
I think technically it was never true. When the tests are run normally, it's now run in parallel and passes. Previously it was run in serial and the test passes.

I moved the functionality in to a branch from a TestMain function, which seems like the more idiomatic way to do this.